components/infobox/wikis/rocketleague/infobox_team_custom.lua

```diff
@@ -91,7 +91,12 @@ function CustomTeam:addToLpdb(lpdbData, args)
 		lpdbData.logo = 'File:' .. args.image
 	end
 
-	lpdbData.extradata = { rating = Variables.varDefault('rating') }
+	local isSchoolTierTeam = string.lower(args.regionsubtier or '') == 'school'
+
+	lpdbData.extradata = {
+		rating = Variables.varDefault('rating'),
+		isschooltierteam = isSchoolTierTeam
+	}
```

Comment:
Are there other regional subtiers, or can there be other regional subtiers in the future?

If there are, or expected to be, isn't it better to store the normalized subtier in lpdb?
If the only subtier is school and no other is expected, is subtier the best name for it?